### PR TITLE
monero 0.17.1.3 -> 0.17.1.5

### PR DESCRIPTION
**Pull request by rnhmjoj**
###### Motivation for this change
Version bump

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s)
- [x] Tested compilation of all pkgs that depend on this change
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Notes: my client stopped synchronising to the remote node, maybe it's unrelated but it would be good if someone could confirm.

### CODE REVIEW

File: pkgs/applications/blockchains/monero-gui/default.nix [hunk 1]
```diff
@@ -59,14 +59,7 @@ stdenv.mkDerivation rec {
     chmod -R +w source/monero
   '';
 
-  patches = [
-    ./move-log-file.patch
-    # fix build failure due to invalid use of CMAKE_PREFIX_PATH
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/applications/blockchains/monero-gui/default.nix:65]: Please remove fetchpatch from inputs

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 106010` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>monero-gui</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>monero</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 106010` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>monero</li>
    <li>monero-gui</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 106010` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>monero</li>
    <li>monero-gui</li>
  </ul>
</details>

**Comment by RaghavSood** [comment]
Any reason why we aren't bumping up to v0.17.1.5 for both?

File: pkgs/applications/blockchains/monero-gui/default.nix (hunk 1 above)
Comment (rnhmjoj) [reply pkgs/applications/blockchains/monero-gui/default.nix:65]: Thank you, fixed.

**Comment by rnhmjoj** [comment]
> Any reason why we aren't bumping up to v0.17.1.5 for both?

Am I missing something? Isn't it what this PR is doing?

**Comment by RaghavSood** [comment]
Result of `nixpkgs-review pr 106010` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>monero</li>
    <li>monero-gui</li>
  </ul>
</details>

**Comment by RaghavSood** [comment]
@rnhmjoj Total brain fart on my end, apologies - thought one of them was going from 0.17.1.3 -> 0.17.1.4.

LGTM

**Review by RaghavSood — APPROVED** [review]

**Pull request merged**